### PR TITLE
Course10

### PR DESCRIPTION
**Pull request by AryavT**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
Javascript tutorial in kannada
### Why is this valuable (or not)?
it will teach people who speak Kannada how to code in java script eventually making their society more advanced
### How do we know it's really free?
its is free on youtube
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by Ajay-singh1 — APPROVED** [review]
LGTM!

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -273,7 +273,9 @@ Online School
 * [Bangla Python Tutorial for Beginners](https://www.youtube.com/playlist?list=PLlBKlxyCgmsCYJLq9qc5QzaU-oBFJN79B) - Niamul Hasan (StartBit)
 * [Python All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVCH8fRfZ8sUuKEQQYIFMjtJ) - Stack Learner
 * [Python Bangla Tutorial 2023](https://www.youtube.com/playlist?list=PLNMnAEqLBwmpR8JDBOEl0jrzmH1vPnO7v) - Hablu Programmer
+* [Python Bangla Tutorial & Data Science Full Course || Full Guidelines for Python, ML, AI, DL & Django](https://www.youtube.com/watch?v=A3FKSsYy0Zg&list=PLKdU0fuY4OFcot0zyVbM1-zKf_eCUK4zQ) - Study Mart
 * [Python Bangla Tutorials for Beginners](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rz5XqMsTk41_j15_6682BN) - Anisul Islam
+
```
Comment (eshellman) [inline courses/free-courses-bn.md:278]:
```suggestion
```

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: courses/free-courses-bn.md [hunk 2]
```diff
@@ -273,6 +273,7 @@ Online School
 * [Bangla Python Tutorial for Beginners](https://www.youtube.com/playlist?list=PLlBKlxyCgmsCYJLq9qc5QzaU-oBFJN79B) - Niamul Hasan (StartBit)
 * [Python All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVCH8fRfZ8sUuKEQQYIFMjtJ) - Stack Learner
 * [Python Bangla Tutorial 2023](https://www.youtube.com/playlist?list=PLNMnAEqLBwmpR8JDBOEl0jrzmH1vPnO7v) - Hablu Programmer
+* [Python Bangla Tutorial & Data Science Full Course || Full Guidelines for Python, ML, AI, DL & Django](https://www.youtube.com/watch?v=A3FKSsYy0Zg&list=PLKdU0fuY4OFcot0zyVbM1-zKf_eCUK4zQ) - Study Mart
```
Comment (rng70) [inline courses/free-courses-bn.md:276]: swap line 276 and 275. linter issue will be solved

**Review by rng70 — CHANGES_REQUESTED** [review]

**Pull request merged**